lib/dsl/exports.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This file defines the exported JDL DSL related APIs
+ * This file will be used as webpack entry point to produce a UMD bundle.
+ */
+const lexer = require('./lexer');
+const api = require('./api');
+const JDLParser = require('./jdl_parser');
+
+const tokens = lexer.tokens;
```

Comment:
cant we just deconstruct these? `const { tokens, JDLLexer } = require('./lexer')` and same for api

Reply:
@MathieuAA does not like deconstruct syntax 😄

Reply:
Yeah I don't but if it's just for that let's go for it...

Reply:
@MathieuAA but why????????? deconstruction is so coool :smile_cat:

Reply:
EcmaScript modules:

```javascript
import {x, y, z} from "./bar"
export const foo = 666;
```

Are already supported in all ever green browsers.
I guess that in a year or two we will have it supported in all LTS versions of node.js
So we could use it directly in node packages without transpiration 😄

Reply:
I left it as is.

You can change this later to whatever you want in a **separate commit**.
I think consistency is important but alos think using newer syntax is important.
so I am **conflicted**

I suggest you make a decision if the upgrade the syntax in the entire project
or just wait until all jhipster supported versions would have the newest import/export syntax 😄